ffmpeg-examples/Makefile

```diff
@@ -14,7 +14,7 @@ ifeq ($(GDB), TRUE)
 	OPTFLAG += -g
 endif
 
-CFLAGS := $(shell pkg-config --cflags $(SHARED_LIBS)) -I.. -Wall -std=c++11 -fopenmp ${OPTFLAG}
```

Comment:
I think we only need to remove it for macOS. Otherwise, it causes the following error for CI:
https://github.com/k2-fsa/sherpa-ncnn/actions/runs/4380400737/jobs/7668276333

```
g++ sherpa-ncnn-ffmpeg.o -I/usr/include/x86_64-linux-gnu -I.. -Wall -std=c++11  -lavdevice -lavformat -lavfilter -lavcodec -lswresample -lswscale -lavutil -L../build/lib -lsherpa-ncnn-c-api -lsherpa-ncnn-core -lkaldi-native-fbank-core -lncnn -lportaudio -lpthread -lm -o sherpa-ncnn-ffmpeg
/usr/bin/ld: ../build/lib/libncnn.a(cpu.cpp.o): undefined reference to symbol 'omp_set_dynamic@@OMP_1.0'
/usr/bin/ld: /lib/x86_64-linux-gnu/libgomp.so.1: error adding symbols: DSO missing from command line
collect2: error: ld returned 1 exit status
```

Reply:
Fixed.